openrtx/include/rtx/OpMode_M17.hpp

```diff
@@ -134,20 +143,34 @@ class OpMode_M17 : public OpMode
      */
     bool compareCallsigns(const std::string& localCs, const std::string& incomingCs);
 
-
-    bool startRx;                      ///< Flag for RX management.
-    bool startTx;                      ///< Flag for TX management.
-    bool locked;                       ///< Demodulator locked on data stream.
-    bool dataValid;                    ///< Demodulated data is valid
-    bool extendedCall;                 ///< Extended callsign data received
-    bool invertTxPhase;                ///< TX signal phase inversion setting.
-    bool invertRxPhase;                ///< RX signal phase inversion setting.
-    pathId rxAudioPath;                ///< Audio path ID for RX
-    pathId txAudioPath;                ///< Audio path ID for TX
-    M17::M17Modulator    modulator;    ///< M17 modulator.
-    M17::M17Demodulator  demodulator;  ///< M17 demodulator.
-    M17::M17FrameDecoder decoder;      ///< M17 frame decoder
-    M17::M17FrameEncoder encoder;      ///< M17 frame encoder
+    /**
+     * Convert OpenRTX latitude/longitude format into M17 format and vice versa.
+     */
+    void rtx_to_q(int32_t* qlat, int32_t* qlon, int32_t lat, int32_t lon);
```

Comment:
These are actually general-purpose conversion functions: please move them to gps.h or utils.h
Also, there is no "OpenRTX latitude/longitude" and "M17" formats. OpenRTX is using fixed point, while M17 is using Q1.24